randovania/games/dread/json_data/Dairon.txt

```diff
@@ -3096,19 +3082,17 @@ Extra - asset_id: collision_camera_039
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
   > Dock to EMMI Zone Exit Southwest
-      All of the following:
-          Morph Ball
-          Grapple Beam or Spider Magnet or Space Jump or Walljump (Beginner)
+      Morph Ball
```

Comment:
Looking at this again, is morph ball necessary to get to the top? We could just replace this with sliding. My only concern is if it's not possible to jump out of the bottom-right pond without gravity, and thus walljump out.

Reply:
While you can, in fact, just use slide, I guess I didn't change it cause you can't get anywhere further without morph anyway. All paths forward are blocked by it (the morph ball launchers, need morph to grab the item in the shortcut room, the tunnel to CU Access, and the tunnel to Exit Northwest). Should it still be changed anyway?